### DISCUSSION TITLE
Logs for WebStomp plugin

### DISCUSSION THREAD

**Discussion by rayate2410** [rabbitmq/rabbitmq-server#8210] [Questions]
We're using rabbitmq WebSTOMP plugin along with normal amqp connections.

Whenever a client connects via amqp - we can see the logs of authentication. Example below:
```
{"time":"2023-05-16 14:54:35.652829+00:00","level":"info","msg":"accepting AMQP connection <0.23955.72> (10.12.128.211:9495 -> 192.168.95.212:5671)","domain":"rabbitmq.connection","pid":"<0.23955.72>"}

{"time":"2023-05-16 14:54:35.656525+00:00","level":"info","msg":"connection <0.23955.72> (10.12.128.211:9495 -> 192.168.95.212:5671): user 'admin' authenticated and granted access to vhost '/'","domain":"rabbitmq.connection","pid":"<0.23955.72>"}
```

But for WebSTOMP connections I don't see any logs. I didn't see any options to enable logging in https://www.rabbitmq.com/web-stomp.html or https://www.rabbitmq.com/logging.html

This is our logging config:
```
log.console = true
log.console.formatter = json
log.console.level = info
```

RabbitMQ Version: 3.11.13

**Comment by michaelklishin** [comment]
Web STOMP connections are handled by the WebSockets server. I suspect the plugin then does not log any events of its own. You are welcome to contribute a PR that would log the same message, assuming that Cowboy provides access to same client information.

@lhoguin where in the WS handler code would be the best place to log such events?

**Reply by lhoguin** [reply]
There's already this: https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbitmq_web_mqtt/src/rabbit_web_mqtt_handler.erl#L269-L270

It's a little complicated to log something on connection accept/close, as early as possible, but we can do it when the Websocket connection is established in https://github.com/rabbitmq/rabbitmq-server/blob/main/deps/rabbitmq_web_mqtt/src/rabbit_web_mqtt_handler.erl#L91-L105

cc @rayate2410
